charts/seed-monitoring/charts/core/charts/prometheus/values.yaml

```diff
@@ -162,17 +174,53 @@ allowedMetrics:
   - process_max_fds
   - process_open_fds
   machineControllerManager:
-  - process_max_fds
-  - process_open_fds
+  - mcm_cloud_api_requests_failed_total
```

Comment:
```suggestion
```
The MCM doesn't initialize this variable, hence maybe not be always available. This could cause a problem with the Grafana dashboards if I had understood it right. 

Something to be fixed at the MCM before exposing it here. So we will have to avoid exposing this metric for now right? Refer - https://github.com/gardener/machine-controller-manager/issues/211#issuecomment-486078446.

Reply:
I combined this metric in a graph together with `mcm_cloud_api_requests_total`. If `mcm_cloud_api_requests_failed_total` is missing the graph will basically not display it. In case of an error the variable will be populated, the metric will appear and the graph will then display the corresponding line. 

Think this is ok for the moment. If this is fixed in the MCM soon, then the line in the graph will be always available.

Reply:
Okay then. We can keep this and shall fix MCM to always display this in the upcoming release.